config_manager.py

```
@@ -0,0 +1,348 @@
+
+            # Write updated file
+            with open(self.env_path, 'w') as f:
+                f.writelines(updated_lines)
+
+            # Update environment variable for current process
+            os.environ[key] = value
+
+            logger.info(f"Updated {key}={value} in .env file")
+
+        except Exception as e:
+            logger.error(f"Failed to update .env file: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)